src/sailor/db/elasticsearch.lua

```diff
@@ -0,0 +1,130 @@
+-------------------------------------------------------------------------------------------
+-- elasticsearch.lua: DB module for connecting, querying and serching through elasticsearch
+-- This file is a part of Sailor project
+-- Copyright (c) 2016 Nikhil. R <rnikhil96@outlook.com>
+-- License: MIT
+-- http://sailorproject.org
+-------------------------------------------------------------------------------------------
+
+
+
+local elasticsearch = require "elasticsearch"
+local main_conf = require "conf.conf"
+local elastic_conf = main_conf.db[main_conf.sailor.search_database]
+
+local elastic = {}
+
+	local client = elasticsearch.client{
+	host={
```

Comment:
It is `hosts` instead of `host`. Also, you can provide an array of hosts instead of a single host.

Refer to the [documentation](http://elasticsearch-lua.readthedocs.io/en/latest/client_configuration/) for details.